Modules/Exercise05/Tests/Unit/Services/OrderServiceTest.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace Tests\Unit\Services;
+
+use Modules\Exercise05\Services\OrderService;
+use Tests\TestCase;
+
+class OrderServiceTest extends TestCase
+{
+    protected $orderService;
+
+    protected function setUp(): void
+    {
+        parent::setup();
+
+        $this->orderService = new OrderService();
+    }
+
+    /**
+     * @dataProvider provider_input_discount
+     */
+    public function test_handlde_discount($detailOrder, $infoBill)
+    {
+        $result = $this->orderService->handleDiscount($detailOrder);
+
+        $this->assertEquals($result, $infoBill);
+    }
+
+    public function provider_input_discount(): array
+    {
+        return [
```

Comment:
Trong provider này nên thêm có 1 case nữa check độ chính xác tới 2 chữ số thập phân vì cái round nó có thêm tham số thứ 2 là `2`
```
$infoBill['price'] = round(($detailOrder['price'] * 80) / 100, 2);
```
Cái này nice to have :smile: